tests/testrelay/docker-compose.yml

```diff
@@ -36,14 +36,30 @@ services:
       - PGDATABASE
       - PGPASSWORD
     restart: unless-stopped
-    command: bash -c "sleep 15 && /opt/ethindex/bin/ethindex runsync --jsonrpc http://node:8545 --waittime 200"
+    command: runsync --jsonrpc http://node:8545 --waittime 200
```

Comment:
Are you sure that you will not need the sleep?

Reply:
It works without sleep because of the `restart: unless-stopped`.

Reply:
okay